exporting/aws_kinesis/aws_kinesis.c

```diff
@@ -175,4 +213,6 @@ void aws_kinesis_connector_worker(void *instance_p)
         break;
```

Comment:
`return` here. Otherwise, a unit test for the worker will break.

Reply:
I changed!